Description: purge references to gnutls/extra.h
 gnutls/extra.h no longer exists.
 .
 libprelude (1.0.1-SprezzOS1) unstable; urgency=low
 .
   * New upstream
Author: Nick Black <nick.black@sprezzatech.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: SprezzOS
Forwarded: no
Last-Update: 2013-03-20

--- libprelude-1.0.1.orig/prelude-admin/server.c
+++ libprelude-1.0.1/prelude-admin/server.c
@@ -39,7 +39,6 @@
 
 #include <gcrypt.h>
 #include <gnutls/gnutls.h>
-#include <gnutls/extra.h>
 
 #include "prelude-client.h"
 #include "prelude-error.h"
--- libprelude-1.0.1.orig/prelude-admin/prelude-admin.c
+++ libprelude-1.0.1/prelude-admin/prelude-admin.c
@@ -57,7 +57,6 @@
 #include <errno.h>
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
-#include <gnutls/extra.h>
 
 #include "common.h"
 #include "config-engine.h"
